paymentsheet/src/test/java/com/stripe/android/paymentsheet/elements/NameConfigTest.kt

```diff
@@ -10,45 +12,13 @@ class NameConfigTest {
     @Test
     fun `verify determine state returns blank and required when empty or null`() {
         Truth.assertThat(nameConfig.determineState(""))
-            .isEqualTo(NameConfig.Companion.Error.BlankAndRequired)
+            .isEqualTo(BlankAndRequired)
```

Comment:
I would think a Name field does not know whether it is required or not - that's known by the Form, because it depends on the payment method type and possibly other elements (e.g. the state of a "save" checkbox element).

Reply:
I agree, I can update that